dev.yml

```
@@ -6,11 +6,17 @@ services:
         environment:
             POSTGRES_PASSWORD: insights
             POSTGRES_USER: insights
+            PGUSER: insights
             POSTGRES_DB: insights
         ports:
             - "5432:5432"
         volumes:
           - ./.unleash/create_unleashdb.sql:/docker-entrypoint-initdb.d/create_unleashdb.sql
+        healthcheck:
+          test: "pg_isready -h db"
+          interval: 5s
```

```suggestion
          interval: 2s
```
Just to make startup a bit quicker